### PR TITLE
Packages/ophidia primitives

### PR DESCRIPTION
**Pull request by SoniaScard**
Fix configure arguments in the Ophidia Primitives package

### CODE REVIEW

File: var/spack/repos/builtin/packages/ophidia-primitives/package.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ophidia-primitives/package.py]:
```suggestion
# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/ophidia-primitives/package.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+from spack.package import *
+
+
+class OphidiaPrimitives(AutotoolsPackage):
+    """Array-based primitives for the Ophidia framework"""
+
+    homepage = "https://github.com/OphidiaBigData/ophidia-primitives"
+    url = "https://github.com/OphidiaBigData/ophidia-primitives/archive/refs/tags/v1.7.1.tar.gz"
+    maintainers = ["eldoo", "SoniaScard"]
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ophidia-primitives/package.py]: @eldoo Are you okay with being added as a maintainer of this package?

**Review by tldahlgren — COMMENTED** [review]
Confirmed the sha256.

**Comment by tldahlgren** [comment]
Hmm.  I'm having problems installing the package but my issues are with a dependency.

File: var/spack/repos/builtin/packages/ophidia-primitives/package.py (hunk 2 above)
Comment (eldoo) [reply var/spack/repos/builtin/packages/ophidia-primitives/package.py]: Yes, that is ok

**Review by eldoo — DISMISSED** [review]

**Review by eldoo — APPROVED** [review]

**Review by tldahlgren — APPROVED** [review]
LGTM

**Pull request merged**